### ISSUE TITLE
Add Radicle -> Track action

### ISSUE THREAD

**Issue by gsaslis** [cytechmobile/radicle-jetbrains-plugin#194]
Collaboration, in Radicle, starts by **tracking** a peer. 

Before we can collaborate with radicle patch proposals and issues, we'll still need to track the peer before we can even learn about the issues or PRs they're opening. 

The track action should obviously ask for the peer id to track and invoke the corresponding CLI command. 
However, it shouldn't stop there. 

It should explain some of the following (or point to a guide that does): 
 
- what does tracking mean in heartwood?
- do I have to track someone to see their patch proposals?
- does tracking mean I always fetch all their branches?
- if I untrack someone, does that delete anything I already have stored?
- do I have to track other delegates in a multi-delegate project ?

**Comment by Stelios123** [comment]
rad track is not yet supported 

![Image](https://user-images.githubusercontent.com/31673488/221853998-a9180dc2-a6fa-4393-844a-5ffab721ed56.png)

**Comment by gsaslis** [comment]
@Stelios123 / @JChrist I've also edited the description here. 

As part of this issue, we should: 
- create a wiki that answers these questions for ourselves
- implement the radicle track action
- figure out what is the jetbrains-native way to provide the necessary help to the users (this needs some more digging and might turn into a separate issue, but please let me know if you already have some idea about this)

**Comment by gsaslis** [comment]
Looking into the `track` command a bit more: 

it is possible to track not just peers, but also repositories, so our UI should support both - and, in doing so, allow: 
- defining an `alias` for the **peer** DID
- define a `scope` for the **repository** being tracked  (see [here for details](https://github.com/radicle-dev/heartwood/blob/414477a31676d5e75efa7f3e8dc6624bcf2b2e52/radicle-cli/src/commands/track.rs#L17) ).

**Comment by Stelios123** [comment]
Ok so I am thinking to add a track button inside radicle tool window (next to the search field) and when the user clicks the button then a dialog will appear where the user can fill did , select the scope, etc..

**Comment by gsaslis** [comment]
@Stelios123 apologies, I wasn't very clear on where the action should be added. 

First, let's add it in the menu - and in the toolbar (exactly where we had the other actions).

**Comment by Stelios123** [comment]
I don't have a track icon, maybe I can use this one? 

![Image](https://github.com/cytechmobile/radicle-jetbrains-plugin/assets/31673488/f6a8f926-978f-4459-9c20-62796ac8e153)

This icon was for publish command I think

**Comment by gsaslis** [comment]
@Stelios123 https://github-production-user-asset-6210df.s3.amazonaws.com/31673488/239250452-f6a8f926-978f-4459-9c20-62796ac8e153.png looks better, no?

**Comment by Stelios123** [comment]
We are using this

**Comment by Stelios123** [comment]
Result : 

https://github.com/cytechmobile/radicle-jetbrains-plugin/assets/31673488/4c908110-1270-4ead-861c-eea88a8af3f8
